### PR TITLE
Add arm64 support for docker builds

### LINKED ISSUE #3336: Introduce aarch64 docker images on dockerhub
## Abstract

Follow up from #3272

## Specification
Provide pre-build aarch64 docker images on docker hub using CI

Check comment for more info: https://github.com/rotki/rotki/issues/2280#issuecomment-843387733

### PR DESCRIPTION
**Pull request by fiahil**
Hello !

This PR is a proposed fix for docker builds on ARM64. I also updated the corresponding GitHub actions for automatically building images for both platforms.

I tried to deploy Rotki on my Homelab running Kubernetes on Raspberry pi. I noticed a few errors when building the docker image for ARM64. Most notable changes includes: 

- Using a common Debian base -sharing the same glic version- in the frontend and backend build stages
- Installing Pyinstaller manually and not through pip (bootloader compilation fails on ARM64)

This patch has been tested locally on ARM64 and AMD64.

Have a great day!

Fix #3336

### CODE REVIEW

**Comment by LefterisJP** [comment]
Thank you @fiahil!

@kelsos is on vacation right now but when he is back he will check this PR.

File: .github/workflows/rotki_nightly_builds.yml [hunk 1]
```diff
@@ -220,8 +220,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
```
Comment (kelsos) [inline .github/workflows/rotki_nightly_builds.yml:228]: This doesn't seem to print anything so maybe we could remove it?

File: .github/workflows/rotki_packaging.yaml [hunk 2]
```diff
@@ -360,8 +360,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
```
Comment (kelsos) [inline .github/workflows/rotki_packaging.yaml:368]: I guess the same goes for this

File: .github/workflows/rotki_nightly_builds.yml (hunk 1 above)
Comment (fiahil) [reply .github/workflows/rotki_nightly_builds.yml:228]: Indeed ! I'll remove them

**Comment by kelsos** [comment]
@fiahil I don't have a way to test the generated docker image unfortunately but it builds properly (although slowly)

The automated build updated the dev tag and now that one has also a linux/arm64 image published. Would it be easy for you to verify that it works properly?

**Review by kelsos — APPROVED** [review]
lgtm on my side

**Comment by fiahil** [comment]
I confirm that it works perfectly :)

**Comment by kelsos** [comment]
> I confirm that it works perfectly :)

Awesome thank you.

**Pull request merged**